__tests__/app.test.js

```diff
@@ -109,6 +109,50 @@ describe("get article by id", () => {
       });
   });
 });
+
+describe("get comments by article id", () => {
+  it("200: returns comments for given id", () => {
```

Comment:
i think this test description and the description on line 135 should be swapped to make more sense for what's being asserted